test/TestHelper.cpp

```diff
@@ -890,4 +901,42 @@ void Listener::notifyTestFinished()
 		g_listener->testFinished();
 }
 
+size_t TestOutputHelper::m_currTest = 0;
+size_t TestOutputHelper::m_maxTests = 0;
+string TestOutputHelper::m_currentTestName = "n/a";
+
+using namespace boost;
+void TestOutputHelper::initTest(json_spirit::mValue& _v)
+{
+	std::string testCaseName = boost::unit_test::framework::current_test_case().p_name;
+	std::cerr << "Test Case \"" + testCaseName + "\": " << std::endl;
```

Comment:
with verbosity 1 only error messages are thrown. But thus it is kind of unclear wethere tests are doing ok, or they stack at some point. Tats why I added output to show that tests are working.